src/search/search_engines/plugin_lazy_wastar.cc

```diff
@@ -1,75 +1,81 @@
-        "is ranked by g + w * h. ");
-    parser.document_note(
-        "Equivalent statements using general lazy search",
-        "\n```\n--evaluator h1=eval1\n"
-        "--search lazy_wastar([h1, eval2], w=2, preferred=h1,\n"
-        "                     bound=100, boost=500)\n```\n"
-        "is equivalent to\n"
-        "```\n--evaluator h1=eval1 --heuristic h2=eval2\n"
-        "--search lazy(alt([single(sum([g(), weight(h1, 2)])),\n"
-        "                   single(sum([g(), weight(h1, 2)]), pref_only=true),\n"
-        "                   single(sum([g(), weight(h2, 2)])),\n"
-        "                   single(sum([g(), weight(h2, 2)]), pref_only=true)],\n"
-        "                  boost=500),\n"
-        "              preferred=h1, reopen_closed=true, bound=100)\n```\n"
-        "------------------------------------------------------------\n"
-        "```\n--search lazy_wastar([eval1, eval2], w=2, bound=100)\n```\n"
-        "is equivalent to\n"
-        "```\n--search lazy(alt([single(sum([g(), weight(eval1, 2)])),\n"
-        "                   single(sum([g(), weight(eval2, 2)]))],\n"
-        "                  boost=1000),\n"
-        "              reopen_closed=true, bound=100)\n```\n"
-        "------------------------------------------------------------\n"
-        "```\n--search lazy_wastar([eval1, eval2], bound=100, boost=0)\n```\n"
-        "is equivalent to\n"
-        "```\n--search lazy(alt([single(sum([g(), eval1])),\n"
-        "                   single(sum([g(), eval2]))])\n"
-        "              reopen_closed=true, bound=100)\n```\n"
-        "------------------------------------------------------------\n"
-        "```\n--search lazy_wastar(eval1, w=2)\n```\n"
-        "is equivalent to\n"
-        "```\n--search lazy(single(sum([g(), weight(eval1, 2)])), reopen_closed=true)\n```\n",
-        true);
+    {
+        parser.document_synopsis(
+            "(Weighted) A* search (lazy)",
+            "Weighted A* is a special case of lazy best first search.");
 
-    parser.add_list_option<shared_ptr<Evaluator>>("evals", "evaluators");
-    parser.add_list_option<shared_ptr<Evaluator>>(
-        "preferred",
-        "use preferred operators of these evaluators", "[]");
-    parser.add_option<bool>("reopen_closed", "reopen closed nodes", "true");
-    parser.add_option<int>("boost",
-                           "boost value for preferred operator open lists",
-                           DEFAULT_LAZY_BOOST);
-    parser.add_option<int>("w", "evaluator weight", "1");
-    SearchEngine::add_succ_order_options(parser);
-    SearchEngine::add_options_to_parser(parser);
-    Options opts = parser.parse();
+        parser.add_list_option<shared_ptr<Evaluator>>(
+            "evals",
+            "evaluators");
+        parser.add_list_option<shared_ptr<Evaluator>>(
+            "preferred",
+            "use preferred operators of these evaluators", "[]");
```

Comment:
Default option should be on a new line (at least I think I saw you do this change on other places).

Reply:
done